IDEAS.md

```diff
@@ -19,6 +21,11 @@ This would be optional and would check if the requirement are fulfilled before e
 ### Website
 Creating a website like [gupax.io](https://gupax.io) to have a more user friendly presentation and installation method.  
 Having a website, we can detect the architecture and OS of the visitor and give them the right archive to download.
+### XMRvsBeast explanation
+Currently in the [README](https://github.com/Cyrix126/gupaxx#what-is-gupaxxxmrvsbeast), there is no explanation on what XMRvsBeast is.
+The README explains what XMRvsBeast it does but not what it is.
+we could improve this with either creating another tab or expanding the [current one](https://github.com/Cyrix126/gupaxx#what-is-gupaxxxmrvsbeast).
+> I think this necessary because noob users have no clue what XvB is. I know that it links to the [XvB website](https://xmrvsbeast.com/p2pool/), but the site would just go right over their heads.
```

Comment:
> I think this necessary because noob users have no clue what XvB is. I know that it links to the [XvB website](https://xmrvsbeast.com/p2pool/), but the site would just go right over their heads.